mindsdb/interfaces/chatbot/memory.py

```diff
@@ -58,7 +58,9 @@ def set_mode(self, chat_id, mode):
 
     def add_to_history(self, chat_id, chat_message):
 
-        return self._add_to_history(chat_id, chat_message)
+        self._add_to_history(chat_id, chat_message)
+        if chat_id in self._cache:
+            del self._cache[chat_id]
```

Comment:
Just to double check: the method does not return anything now, is this intended?

Reply:
yes, it used to return value but the output isn't used anywhere